src/rgw/rgw_op.cc

```diff
@@ -3362,6 +3371,11 @@ int RGWPutObj::init_processing(optional_yield y) {
 
 int RGWPutObj::verify_permission(optional_yield y)
 {
+  #ifdef HAVE_JAEGER
+  auto verify_permission_span = rgw_tracer->start_span("verify_permission", this->trace);
+  verify_permission_span->SetTag(tracing::USER_DISPLAY_NAME, s->user->get_display_name());
+  verify_permission_span->SetTag(tracing::BUCKET_NAME, s->bucket->get_name());
```

Comment:
not sure which tags are fully generic.
do we have a bucket name in each request?

Reply:
done in 2947371
now we add those generic tags : bucket_name, user_id, object_name, op and return
and upload_id in RGWInitMultipart , RGWCompleteMultipart and in 'RGWPutObj` in case it's in multipart mode.